lib/ex_prosemirror/block/heading.ex

```diff
@@ -20,6 +20,7 @@ defmodule ExProsemirror.Block.Heading do
   import Ecto.Changeset
 
   alias ExProsemirror.Block.Text
+  alias ExProsemirror.HTML.Encoder
```

Comment:
Maybe it's better you alias it for 
```suggestion
  alias ExProsemirror.HTML.Encoder, as: HtmlEncoder
```
Then you can have many tipes of encoders without needing to change anything in the future.